core/notifications/notification_store.go

```diff
@@ -0,0 +1,81 @@
+package notifications
+
+import (
+	"github.com/anyproto/any-sync/app"
+	"github.com/dgraph-io/badger/v4"
+	"github.com/gogo/protobuf/proto"
+	ds "github.com/ipfs/go-datastore"
+
+	"github.com/anyproto/anytype-heart/pkg/lib/datastore"
+	"github.com/anyproto/anytype-heart/pkg/lib/localstore"
+	"github.com/anyproto/anytype-heart/pkg/lib/pb/model"
+	"github.com/anyproto/anytype-heart/util/badgerhelper"
+)
+
+const (
+	notificationsPrefix   = "notifications"
+	notificationStoreName = "notification_store"
+)
+
+var notificationsInfo = ds.NewKey("/" + notificationsPrefix + "/info")
+
+type NotificationStore interface {
+	app.Component
```

Comment:
seems like notification store could be just an ordinary object without properties of a service, it could be initialized inside notification service